ledger/daml-on-sql/src/main/scala/on/sql/Cli.scala

```diff
@@ -43,6 +43,12 @@ private[sql] final class Cli(
       )
       .action((url, config) => config.copy(jdbcUrl = Some(url)))
 
+    parser
+      .opt[Int]("database-connection-pool-size")
+      .optional()
+      .text("The number of connections in the database connection pool.")
```

Comment:
Perhaps include the default in the text.

Reply:
Done